src/index.scss

```diff
@@ -7,34 +7,14 @@ body {
   -webkit-font-smoothing: antialiased;
   -moz-osx-font-smoothing: grayscale;
   color: $light;
+  overflow-x: hidden;
```

Comment:
You can remove this as someone else is solving this issue. And also this is not the ideal way of solving the overflow issue. The overflow is happening because of the image's size so giving it a max-width would solve the issue.
Refer to https://github.com/ChoukseyKhushbu/Gitzilla/pull/22 for that